src/components/layout/footer/common/style.js

```diff
@@ -187,7 +192,7 @@ export const BoldLink = styled(StyledLink)`
     color: var(--color-black-3);
     font-size: var(--text-size-xs);
     @media ${device.tabletL} {
-        font-size: var(--text-size-sm);
+        font-size: 12px;
```

Comment:
Can we add this in the `variable.js` so it's reusable. @sean-binary  do you think we should start updating the global variable?  I can see alot of `font-sizes` that can be optimized by using the reusable variables

Reply:
Yes that would be the ideal case, but currently you'll find a lot of cases where desktop font size will be in `rem` for e.g. to so that it scales up consistently above 1980px, and `px` for < tabletL sizes. This will be quite difficult to replicate for CSS variables use case.

I think before that, we need to remove

```css
    html {
        font-size: 62.5%; /* 1rem = 10px */
        box-sizing: border-box;

        @media ${device.desktopL} {
            font-size: 75.5%; /* 1rem = 12px */
        }
        @media ${device.laptop} {
            font-size: 58%; /* 1rem = 9px */
        }
        @media ${device.tabletL} {
            font-size: 50%; /* 1rem = 8px */
        }
    }
```

so that it's consistent 1rem = 10px. Maybe we can draw out a plan for this. @prince-deriv

Reply:
Okay we can have a discussion for that. But in this case can we allow this `12px` for now?

Reply:
Yep should be fine 👍